content/updates/2018-02.md

```diff
@@ -7,7 +7,7 @@ image: https://266-97323138-gh.circle-artifacts.com/0/pi-image.img.xz
 
 Vision:
 
-- Completely re-do vision to greatly increase accuracy.
+- Completely re-do vision to greatly increase accuracy. Now all Cartesian values have an error of less than 15% of the total distance.
```

Comment:
Describing this as a "tolerance" rather than an "error" might be clearer? (The sentence as a whole might need re-wording if we do that though.)

Reply:
Fixed in 59a1773